widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -230,6 +269,10 @@ export function ConfirmWalletsModal(props: PropTypes) {
     );
   }, [connectedWallets.length]);
 
+  useEffect(() => {
+    setIsFirefox(navigator?.userAgent.includes('Firefox'));
+  }, []);
+
```

Comment:
Is the "isFirefox" state and this "useEffect" necessary? Can we do without them?
`const isFirefox = navigator?.userAgent.includes('Firefox')`

Reply:
You are right. Fixed.